src/Block/Js.php

```diff
@@ -45,32 +63,40 @@ public function getOfflinePageUrl()
     }
 
     /**
-     * Get the list of URLs unavailable for caching or viewing offline.
+     * Get the list of URLs for external scripts to import into the service worker.
      *
-     * @return array
+     * @return string[]
      */
-    public function getUrlBlacklist()
+    public function getExternalScriptUrls()
     {
-        return $this->config->getUrlBlacklist();
+        $scripts = [
+            $this->getViewFileUrl("Meanbee_ServiceWorker::js/lib/workbox-sw.prod.v1.0.1.js"),
+        ];
+
+        if ($this->isGaOfflineEnabled()) {
+            $scripts[] = $this->getViewFileUrl("Meanbee_ServiceWorker::js/lib/workbox-google-analytics.prod.v1.0.0.js");
```

Comment:
How would you validate it?

`$this->getViewFileUrl()` always returns a URL (asset url, if it exists; not found url, if it doesn't)